tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml-indexed-access-negative.bal

```diff
@@ -19,3 +10,27 @@ function testNonStringIndexAccess() {
     var k = x[true];
     var b = x[1.2];
 }
+
+function testInvalidXMLAccessWithIndex() {
+    xml x1 = xml `<root><!-- comment node--><name>supun</name><city>colombo</city></root>`;
+    xml x2 = x1[0]/*;
+
+    x2[1] = xml `<address/>`;
+
+    string|'xml:Text x3 = xml `sample test`;
+    string sampleString1 = x3[0];
+    xml x4 = x3[0];
+
+    'xml:Element x5 = xml `<root><!-- comment node--><name>anne</name><city>colombo</city></root>`;
+    xml x6 = x5[0]/*;
+    'xml:Comment x7 = xml `<!-- comment node-->`;
+    xml x8 = x7[0]/*;
+    'xml:ProcessingInstruction x9 = xml `<?xml-stylesheet href="mystyle.css" type="text/css"?>`;
+    xml x10 = x9[0]/*;
+
+    x6 = x5[0][0];
+    x8 = x7[0][0];
+    x10 = x9[0][0];
+
+    int i = 'xml:length(x5[0]);
```

Comment:
The error message doesn't feel right for lang-lib invocation with function invocation syntax. I think this is good when langlib function is invoked with method invocation syntax.

Reply:
We also need to create an issue to track enabling this support